include/swift/AST/DiagnosticsDriver.def

```diff
@@ -88,7 +88,7 @@ WARNING(warning_unnecessary_repl_mode,none,
         "unnecessary option '%0'; this is the default for '%1' "
         "with no input files", (StringRef, StringRef))
 ERROR(error_unsupported_option,none,
-      "unsupported option '%0' for '%1'; did you mean '%2 %0'?",
+      "the option '%0' is not supported by '%1'; did you mean to use '%2'?",
```

Comment:
I've made an update removing the "the". Thanks for the link! I will make sure to do some reading in the docs directory before my next change 😄.